src/pages/index.js

```diff
@@ -1,245 +1,316 @@
+              <>
+                <h4>Untagged Pages</h4>
+                <p>
+                  Total: <strong>{unpubPagesUntagged.length}</strong>
+                </p>
+                <ul className="three-col-md">
                   {unpubPagesUntagged.map((page) => (
-                      <li key={page.node.drupal_id}><Link to={page.node.path.alias}>{page.node.title}</Link></li>
+                    <li key={page.node.drupal_id}>
+                      <Link to={page.node.path.alias}>{page.node.title}</Link>
+                    </li>
                   ))}
-              </ul>
-              </>}              
-                            
-              {unpubPrograms.length > 0 && <>
+                </ul>
+              </>
+            )}
+
+            {unpubPrograms.length > 0 && (
+              <>
                 <h3>Programs</h3>
-                <p>Total: <strong>{unpubPrograms.length}</strong></p>
+                <p>
+                  Total: <strong>{unpubPrograms.length}</strong>
+                </p>
                 <ul className="three-col-md">
-                    {unpubPrograms.map((program) => (
-                        <li key={program.node.drupal_id}><Link to={program.node.path.alias}>{program.node.title}</Link></li>
-                    ))}
+                  {unpubPrograms.map((program) => (
+                    <li key={program.node.drupal_id}>
+                      <Link to={program.node.path.alias}>{program.node.title}</Link>
+                    </li>
+                  ))}
                 </ul>
-              </>}
-            </div>
+              </>
+            )}
           </div>
         </div>
+      </div>
     </Layout>
-    )
-}
+  );
+};
 
-export default IndexPage
+export default IndexPage;
 
-export const query = graphql`{
-  accordion: homeYaml(yamlId: {eq: "home_accordion"}) {
-    id
-    accordion {
-      title
-      content
-    }
-  }
-  allNodePage(sort: {title: ASC}) {
-    edges {
-      node {
+export const query = graphql`
+  {
+    accordion: homeYaml(yamlId: { eq: "home_accordion" }) {
+      id
+      accordion {
         title
-        drupal_id
-        path {
-          alias
+        content
+      }
+    }
+    pubPages: allNodePage(filter: { moderation_state: { eq: "published" } }, sort: { title: ASC }) {
+      edges {
+        node {
+          title
+          drupal_id
+          id
+          path {
+            alias
+          }
+          relationships {
+            field_tags {
+              __typename
+              ... on TaxonomyInterface {
+                drupal_id
+                id
+                name
+              }
+            }
+          }
         }
-        relationships {
-          field_tags {
-            __typename
-            ... on TaxonomyInterface {
-              drupal_id
-              id
-              name
+      }
+    }
+    unpubPages: allNodePage(filter: { moderation_state: { eq: "draft" } }, sort: { title: ASC }) {
```

Comment:
On the index page, I'm seeing the (null) Alias Test in the list, but it does not link anywhere.